data/data-pipeline/data_pipeline/ipython/scoring_comparison.ipynb

```diff
@@ -214,35 +207,60 @@
     "CALENVIROSCREEN_PERCENTILE_FIELD = \"calenviroscreen_percentile\"\n",
     "CALENVIROSCREEN_PRIORITY_COMMUNITY_FIELD = \"calenviroscreen_priority_community\"\n",
     "\n",
-    "calenviroscreen_data_path = (\n",
-    "    DATA_DIR / \"dataset\" / \"calenviroscreen4\" / \"data06.csv\"\n",
-    ")\n",
+    "calenviroscreen_data_path = DATA_DIR / \"dataset\" / \"calenviroscreen4\" / \"data06.csv\"\n",
     "calenviroscreen_df = pd.read_csv(\n",
     "    calenviroscreen_data_path,\n",
     "    dtype={ExtractTransformLoad.GEOID_TRACT_FIELD_NAME: \"string\"},\n",
     ")\n",
     "\n",
     "# Convert priority community field to a bool.\n",
-    "calenviroscreen_df[\n",
+    "calenviroscreen_df[CALENVIROSCREEN_PRIORITY_COMMUNITY_FIELD] = calenviroscreen_df[\n",
     "    CALENVIROSCREEN_PRIORITY_COMMUNITY_FIELD\n",
-    "] = calenviroscreen_df[CALENVIROSCREEN_PRIORITY_COMMUNITY_FIELD].astype(bool)\n",
+    "].astype(bool)\n",
     "\n",
     "calenviroscreen_df.head()"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b1ac2854-80c8-42a8-85e8-84c5684bbe43",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "## *I THINK* it makes sense to pull this into \"code code\" at least so it can read the same sort of stuff????\n",
+    "\n",
+    "# Mapping for EJ\n",
+    "\n",
+    "MAPPING_FOR_EJ_FINAL_RANK_FIELD = field_names.MAPPING_FOR_EJ_FINAL_RANK_FIELD\n",
+    "MAPPING_FOR_EJ_FINAL_SCORE_FIELD = field_names.MAPPING_FOR_EJ_FINAL_SCORE_FIELD\n",
+    "MAPPING_FOR_EJ_PRIORITY_COMMUNITY_FIELD = (\n",
+    "    field_names.MAPPING_FOR_EJ_PRIORITY_COMMUNITY_FIELD\n",
+    ")\n",
+    "\n",
+    "mapping_for_ej_path = DATA_DIR / \"dataset\" / \"mapping_for_ej\" / \"co_va.csv\"\n",
+    "mej_df = pd.read_csv(\n",
```

Comment:
This is a nitty comment but I usually find acronyms and shortenings to be more trouble than they're worth -- so maybe move to just calling something like `mapping_for_ej_df`?